src/libraries/System.Runtime/tests/System.Globalization.Tests/CompareInfo/CompareInfoTests.HashCode.cs

```diff
@@ -12,46 +12,23 @@ namespace System.Globalization.Tests
 {
     public class CompareInfoHashCodeTests : CompareInfoTestsBase
     {
-        public class CustomComparer : StringComparer
-        {
-            private readonly CompareInfo _compareInfo;
-            private readonly CompareOptions _compareOptions;
-
-            public CustomComparer(CompareInfo cmpInfo, CompareOptions cmpOptions)
-            {
-                _compareInfo = cmpInfo;
-                _compareOptions = cmpOptions;
-            }
-
-            public override int Compare(string x, string y) =>
-                _compareInfo.Compare(x, y, _compareOptions);
-
-            public override bool Equals(string x, string y) =>
-                _compareInfo.Compare(x, y, _compareOptions) == 0;
-
-            public override int GetHashCode(string obj)
-            {
-                return _compareInfo.GetHashCode(obj, _compareOptions);
-            }
-        }
-
         public static IEnumerable<object[]> HashCodeLocalized_TestData()
         {
             yield return new object[] { s_invariantCompare, "foo", "Foo", CompareOptions.IgnoreCase };
-            yield return new object[] { s_invariantCompare, "igloo", "\u0130GLOO", CompareOptions.IgnoreCase }; // FAILS
+            yield return new object[] { s_invariantCompare, "igloo", "\u0130GLOO", CompareOptions.IgnoreCase };
+            yield return new object[] { s_invariantCompare, "igloo", "\u0130GLOO", CompareOptions.None };
             yield return new object[] { s_invariantCompare, "igloo", "IGLOO", CompareOptions.IgnoreCase };
-            yield return new object[] { new CultureInfo("pl-PL").CompareInfo, "igloo", "\u0130GLOO", CompareOptions.IgnoreCase }; // FAILS
+            yield return new object[] { new CultureInfo("pl-PL").CompareInfo, "igloo", "\u0130GLOO", CompareOptions.IgnoreCase };
             yield return new object[] { new CultureInfo("pl-PL").CompareInfo, "igloo", "IGLOO", CompareOptions.IgnoreCase };
             yield return new object[] { new CultureInfo("tr-TR").CompareInfo, "igloo", "\u0130GLOO", CompareOptions.IgnoreCase };
-            yield return new object[] { new CultureInfo("tr-TR").CompareInfo, "igloo", "IGLOO", CompareOptions.IgnoreCase }; // FAILS
+            yield return new object[] { new CultureInfo("tr-TR").CompareInfo, "igloo", "IGLOO", CompareOptions.IgnoreCase };
 
             if (!PlatformDetection.IsHybridGlobalizationOnBrowser)
             {
                 // ActiveIssue: https://github.com/dotnet/runtime/issues/96400
```

Comment:
https://github.com/dotnet/runtime/issues/96400 is already closed , do we need this comment here?